secure-storage/secure-storage-impl/src/test/java/com/duckduckgo/securestorage/impl/RealSecureStorageKeyGeneratorTest.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.securestorage.impl
+
+import com.duckduckgo.appbuildconfig.api.AppBuildConfig
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.mock
+import org.mockito.MockitoAnnotations
+import org.mockito.kotlin.any
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import java.security.Key
+
+class RealSecureStorageKeyGeneratorTest {
+    @Mock
+    private lateinit var appBuildConfig: AppBuildConfig
+
+    @Mock
+    private lateinit var derivedKeySecretFactory: DerivedKeySecretFactory
+
+    @Mock
+    private lateinit var legacyDerivedKeySecretFactory: DerivedKeySecretFactory
+
+    private lateinit var testee: RealSecureStorageKeyGenerator
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.openMocks(this)
+
+        val key = mock(Key::class.java)
+        whenever(key.encoded).thenReturn("Zm9vYg==".toByteArray())
```

Comment:
Would be good to pull this `Zm9vYg==` into a variable since it's accessed a lot in this class, and it reads like a magic number (well, string)